### DISCUSSION TITLE
Breathing noise in inference with XTTS-v2

### DISCUSSION THREAD

**Discussion by ukemamaster** [coqui-ai/TTS#3254] [General Q&A]
Inference with XTTS-v2 original model tends to have breathing noises every now and then (mostly after a comma or full stop in the text) in the output. For example at 2s, 10s and 13s mark in the following inference. 

https://github.com/coqui-ai/TTS/assets/59258087/04c7be55-7503-4fe8-90b0-e2598c93fec8

It is assumed [here](https://github.com/coqui-ai/TTS/issues/3236#issuecomment-1814132445) that this effect is coming from the training data. 

Is there a way to suppress these noises from my training data so that i can fine tune the model with clean data?

**Comment by marioguima** [comment]
I did a test here and I didn't hear that breathing. Can you show how you are using xtts and include your reference audio so I can play it here and try to help you?

Ps. for this example I'm using this code

`
import torch
from TTS.api import TTS

device = "cuda" if torch.cuda.is_available() else "cpu"

tts = TTS("tts_models/multilingual/multi-dataset/xtts_v2").to(device)

tts.tts_to_file(text="Have you ever wondered why, at birth, we are considered the miracle of life, but when we grow up, we often content ourselves with the common? The philosopher Marcus Aurelius believed that each one of us has an extraordinary purpose\nSo, how do you wake up every day? Does the way you start your morning dramatically affect your success levels?",
                language="en", speaker_wav="voices/EL-01.wav", file_path="output-en-09.wav")
`

https://github.com/coqui-ai/TTS/assets/4010143/552d1863-3f83-47ff-ab6b-cee88f90268d

**Comment by AI-Guru** [comment]
This seems to happen with rather long sentences. And I have to admit, that I like it.

I wonder, is there a way to FORCE a breathing sound here and there? Like forcing a breath-in at the beginning of a sentence?
